nym-vpn/ui/src/constants.ts

```diff
@@ -8,3 +10,5 @@ export const routes = {
 export const AppName = 'NymVPN';
 export const ConnectionEvent = 'connection-state';
 export const ProgressEvent = 'connection-progress';
+//putting this here for now until decided how default country is determined
+export const quickConnectCountry: Country = { name: 'Germany', code: 'DE' };
```

Comment:
for naming consistency sake, global constant should start with a capital letter :p